metricbeat/module/kibana/stats/stats_test.go

```diff
@@ -98,3 +105,68 @@ func TestFetchNoExcludeUsage(t *testing.T) {
 	// First fetch
 	mbtest.ReportingFetchV2Error(f)
 }
+
+func kibanaServer(t *testing.T, path string, version string) *httptest.Server {
```

Comment:
I might steal the pattern from https://github.com/elastic/beats/blob/5ee1964c64382f5f04cb645b13824cc546f19fbe/metricbeat/module/rabbitmq/mtest/server.go#L43 if I continue this effort.